src/cpuinfo_x86.c

```diff
@@ -125,6 +125,41 @@ static bool HasTmmOsXSave(uint32_t xcr0_eax) {
                                MASK_ZMM16_31 | MASK_XTILECFG | MASK_XTILEDATA);
 }
 
+static bool HasSecondFMA(uint32_t model) {
+    // Skylake server
+    if (model==0x55) {
+        char proc_name[49] = {0};
+        FillX86BrandString(proc_name);
+        // detect Xeon
+        if (proc_name[9]=='X') {
+            // detect Silver or Bronze
+            if (proc_name[17]=='S' || proc_name[17]=='B') {
+                return false;
+            }
+            // detect Gold 5_20 and below, except for Gold 53__
+            else if (proc_name[17]=='G' && proc_name[22]=='5') {
```

Comment:
`else` here and below are not necessary

Reply:
Sure, I just assumed if-elif-elif-elif was going to generate equal or better code than if-if-if-if but I am not a compiler person.